src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/client/FuncTestRequestHandler.java

```diff
@@ -54,6 +54,7 @@ public void sendPost(Object data, HttpStatus expectedStatus, String path) throws
                 path);
     }
 
+
     public Response sendPost(String jsonBody, HttpStatus expectedStatus, String path) {
```

Comment:
NB this Jira is a placeholder for running functional tests against new features not available in aat.